trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Helper_Functions/oswald_efficiency.py

```diff
@@ -0,0 +1,62 @@
+## @ingroup Methods-Aerodynamics-Common-Fidelity_Zero-Helper_Functions
+# Oswald_efficiency.py
+# 
+# Created:  Nov 2021, S. Karpuk
+# Modified: 
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+# SUAVE Imports
+import SUAVE
+from SUAVE.Core import Units
+import numpy as np
+
+# ------------------------------------------------------------------------------------
+#  Compute Oswald efficiency using the method of Scholz for the constraint analysis
+# ------------------------------------------------------------------------------------
+
+## @ingroup Methods-Aerodynamics-Common-Fidelity_Zero-Helper_Functions
+def oswald_efficiency(vehicle,cdmin):
+    """Calculate an average Oswald efficiencies based on the method of Scholz for the constraint analysis
+
+        Assumptions:
+        None
+
+        Source:
+           M. Nita, D.Scholtz, 'Estimating the Oswald factor from basic aircraft geometrical parameters',
+            Deutscher Luft- und Raumfahrtkongress 20121DocumentID: 281424
+
+        Inputs:
+            constraint_analysis.geometry.taper                 [Unitless]
+                                aspect_ratio                   [Unitless]
+                                sweep_quarter_chord            [radians]
+                                aerodynamics.fuselage_factor   [Unitless]
+                                viscous_factor                 [Unitless]
+
+        Outputs:
+            e          [Unitless]
+
+        Properties Used:
+
+    """  
+
+    # Unpack inputs
+    taper = vehicle.wings['main_wing'].taper
```

Comment:
I would prefer if this parsed for the main wing rather than look for a name. The code is simple:

from SUAVE.Components.Wings.Main_Wing

main_wing = None
for wing in wings:
	if isinstance(wing,main_wing):
		main_wing = wing